tasks/main.yml

```diff
@@ -13,15 +13,24 @@
         state=directory
   tags: ["config"]
 
-- name: Ensure Spark log and run directories exist
+- name: Ensure Spark log, run, and work directories exist
   file: path="{{ item }}"
         owner={{ spark_user }}
         group={{ spark_user }}
-        mode=0755
+        mode=0775
         state=directory
   with_items:
     - "{{ spark_log_dir }}"
     - "{{ spark_run_dir }}"
+    - "{{ spark_work_dir }}"
```

Comment:
This addition appears to be causing a problem when provisioning from scratch. Because a directory is being created inside of `/usr/lib/spark`, the first item in `Setup Spark distribution symlinks` is failing.